File: 7 kyu/Vowel Count/solution.java
public class Vowels {

    public static int getCount(String str) {
        int vowelsCount = 0;
        String vowels = "aeiou";
        for(int i = 0; i < str.length(); i++)
        {
            if(vowels.contains("" + str.charAt(i)))
                vowelsCount++;
        }
        return vowelsCount;
    }

}

File: 7 kyu/Vowel Count/solution.cs
using System;

public static class Kata
{
    public static int GetVowelCount(string str)
    {
        int vowelCount = 0;
        string vowels = "aeiou";

        for (int i = 0; i < str.Length; i++)
        {
            if (vowels.Contains(str[i]))
            {
                vowelCount += 1;
            }
        }

        return vowelCount;
    }
}
